### PR TITLE
Highlight 'Stick' in Werewolf Agility Course

### LINKED ISSUE #6474: Werewolf Agility Course Highlight stick (and maybe npc at the end)
Honestly i know it might sound dumb, but when i'm not paying the best of attention on the werewolf course, i often forget to pick up the stick, yeah i know it might be dumb but it does happen. Maybe it would be possible to make it similar to marks of grace, where it highlights it on the floor if you haven't picked it up yet and makes the next obstacle red untill you pick it up?

Also this is something i haven't seen before so i'm not sure about the possibilities, but maybe a reminder in some shape of form that you have to turn the stick in? Maybe it's possible to highlight the npc when you have a certain item, i'm not sure about that but I do believe highlighting the stick would basically be exactly the same as the marks of grace, but maybe i'm wrong.

### PR DESCRIPTION
**Pull request by AaronPoon**
Similar to highlighting 'Mark of Grace' when doing Rooftop Agility Courses, it would be helpful to highlight the 'Stick' when doing Werewolf Agility Course.

Closes #6474

### CODE REVIEW

**Comment by KasparRosin** [comment]
This would close #6474

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java [hunk 1]
```diff
@@ -174,4 +174,26 @@ default boolean showAgilityArenaTimer()
...
+			description = "Enable/disable the highlighting of the retrievable Stick in Werewolf Agility Course",
+			position = 13
+	)
+	default boolean highlightStick()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "stickHighlight",
```
Comment (KasparRosin) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java]: maybe should rename to `stickHighlightColor` to match the name and description?

**Comment by dekvall** [comment]
I have made the requested changes on this

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java [hunk 2]
```diff
@@ -122,21 +124,29 @@ public Dimension render(Graphics2D graphics)
...
+			highlightTile(graphics, playerLocation, stickTile, config.getStickColor());
+		}
+
 		return null;
 	}
+
+	private void highlightTile(Graphics2D graphics, LocalPoint playerLocation, Tile tile, Color color)
+	{
+		if (tile.getPlane() == client.getPlane() && tile.getItemLayer() != null
+				&& tile.getLocalLocation().distanceTo(playerLocation) < MAX_DISTANCE)
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java]:
```suggestion
			&& tile.getLocalLocation().distanceTo(playerLocation) < MAX_DISTANCE)
```

**Comment by Nightfirecat** [comment]
Marks of grace are still highlighted properly, and the werewolf course stick highlighting works fine as well.

**Pull request merged**